e2e/test/scenarios/binning/binning-reproductions.cy.spec.js

```diff
@@ -110,6 +110,7 @@ describe("binning related reproductions", () => {
     H.popover()
       .findByRole("option", { name: "CREATED_AT" })
       .findByLabelText("Temporal bucket")
+      .realHover()
```

Comment:
since the fix is to properly hide the element until hover, we need to hover on the element before it's visible. Otherwise, Cypress couldn't click it.